modules/pel/peltool/peltool.py

```diff
@@ -365,14 +369,33 @@ def listOption(path: str, config: Config, extension: str, rev: bool =False):
             file_list.append(file)  ## create list of file names
         # Only process top level directory
         break
-    file_list.sort(reverse=rev) 
+    file_list.sort(reverse=rev)
+    return root, file_list
+
+def listOption(path: str, config: Config, extension: str, rev: bool = False):
+    root, file_list = getFileList(path, extension, rev)
     final_summary = {}
     for file in file_list:
         eid, summary = extractAndSummarizePEL(os.path.join(root, file), config)
         if eid :
             final_summary[eid] = summary
     print(prettyPrint(json.dumps(final_summary, indent=4) , desiredSpace = 29))
 
+def extractAllPELsData(path: str, config: Config, extension: str, rev: bool = False):
+    root, file_list = getFileList(path, extension, rev)
+    allPELsData = "[ \n"
+    for file in file_list:
```

Comment:
Contrary to what the 'all' name implies, we don't actually want all PELs here.  Just ones that aren't informational severity and don't have the hidden action flag set.

before we had the -f and -h options separate to pick those up, but since 99% of informational errors are hidden anyway, it wasn't that useful having separate flags.   So probably better is a new option that is really 'actually all of them'.  Though, for the sake of compatibility we may still want the -f and -h to start with as well.

Reply:
though maybe we keep everything consistent and just use the isServiceable() check.

Reply:
Keeping everything default, just added isServiceable() check.